src/basic/format-util.h

```diff
@@ -71,8 +72,15 @@ typedef enum {
         FORMAT_BYTES_TRAILING_B  = 1 << 2,
 } FormatBytesFlag;
 
-#define FORMAT_BYTES_MAX 8
+#define FORMAT_BYTES_MAX 16
```

Comment:
Why this change?

Reply:
"infinity\0" is 9 bytes. We can special case it for this one case, but it makes it harder to reuse buffers if desired which kind of sucks. Either way, doesn't seem a massive deal.

Reply:
OK.